persistence/binary-android/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>one.microstream</groupId>
+		<artifactId>microstream-persistence-parent</artifactId>
+		<version>07.00.00-MS-GA-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<artifactId>microstream-persistence-binary-android</artifactId>
+	
+	<name>MicroStream Persistence Android</name>
+	<description>MicroStream Persistence Specialized Type Handlers</description>
```

Comment:
MicroStream Android Persistence Specialized Type Handlers
It might be better.

Reply:
Makes sense, I changed it.